yak/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ if(NOT DEFINED CMAKE_CUDA_STANDARD)
   set(CMAKE_CUDA_STANDARD_REQUIRED True)
 endif()
 
-find_package(CUDA 9.0 REQUIRED)
+find_package(CUDA REQUIRED)
```

Comment:
I think this should stay as `find_package(CUDA 9.0 REQUIRED)` to keep the same behavior as before. According to the [CMake docs](https://cmake.org/cmake/help/v3.5/command/find_package.html) `find_package` uses the version number as the minimum required version by default. If we delete the minimum-required version then if someone tries to build using CUDA 8.0 (which isn't supported), they'll get a whole mess of CUDA and C++ compilation errors instead of a message saying "hey your version of CUDA is too old for this package".

Reply:
Understood